wiki/Gameplay/Drain Time/en.md

```diff
@@ -0,0 +1,7 @@
+# Drain Time
```

Comment:
```suggestion
# Drain time
```

Reply:
I thought first-level headers were fine to capitalize all the way through? want a bit of clearance on that

Reply:
Article titles are now going to use sentence case. See #2580 for discussion.